File: pwa/PwaClient/Models/Gpu.cs
public class Gpu
{
    public string Name { get; set; }
    public int VideoRamCapacity { get; set; }

    // Konstruktor
    public Gpu(string name, int videoRamCapacity)
    {
        this.Name = name;
        this.VideoRamCapacity = videoRamCapacity;
    }
}

File: mobile_client/lib/models/gpu.dart
import 'dart:convert';

List<Gpu> gpusFromJson(String str) => List<Gpu>.from(json.decode(str).map((x) => Gpu.fromJson(x)));

class Gpu {
  final String name;
  final int videoRamCapacity;

  Gpu({required this.name, required this.videoRamCapacity});

  factory Gpu.fromJson(Map<String, dynamic> json) => Gpu(
        name: json["name"],
        videoRamCapacity: json["videoRamCapacity"],
      );

  Map<String, dynamic> toJson() => {
        "name": name,
        "videoRamCapacity": videoRamCapacity,
      };
}